src/bisect.jl

```diff
@@ -0,0 +1,42 @@
+
+const where_bisect = 0.49609375
```

Comment:
This looks like a magic constant :) 0.5 - 1/256, if I'm not mistaken. Is there a particular reason for that?